### PR TITLE
Build using the JSON license list on spdx.org

### PR DESCRIPTION
**Pull request by kemitchell**
Hello again!

This pull request uses the preview JSON SPDX license identifier list on spdx.org instead of `spdx-license-list`.

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -9,26 +10,36 @@ const stringifyObject = require('stringify-object');
...
-];
+http.get('http://spdx.org/licenses/licenses.json', response => {
+  let buffer = '';
+  response
+    .on('data', data => {
+      buffer += data;
+    })
+    .on('end', () => {
+      const json = JSON.parse(buffer);
+      let array = json.licenses.map(license => license.licenseId).sort();
```
Comment (shinnn) [inline build.js]: Do you think this `.sort()` is needed? In the current version I respect the original order.

File: README.md [hunk 2]
```diff
@@ -52,4 +52,4 @@ var spdxLicenseIds = require('spdx-license-ids'); //=> ['Glide', 'Abstyles', 'AF
 
 ## License
 
-[The Uicense](./LICENSE).
+[The Unlicense](./LICENSE).
```
Comment (shinnn) [inline README.md]: Thanks :)

File: build.js (hunk 1 above)
Comment (kemitchell) [inline build.js]: I don't believe the order of the identifiers has any significance.

If anything, I hoped sorting would keep the diff short if and when you run the script to update the JSON files.

File: build.js [hunk 3]
```diff
@@ -9,26 +10,36 @@ const stringifyObject = require('stringify-object');
...
-  }
-];
+http.get('http://spdx.org/licenses/licenses.json', response => {
+  let buffer = '';
+  response
+    .on('data', data => {
+      buffer += data;
+    })
+    .on('end', () => {
+      const json = JSON.parse(buffer);
```
Comment (sindresorhus) [inline build.js]: This code could be a lot simpler if you used [`got`](https://github.com/sindresorhus/got) ;)

Comment (kemitchell) [inline build.js]: I hear [it's even easier in Ruby](https://github.com/kemitchell/rubygems/commit/a2630898bdb9cfab4d3ef6965112644276639740#diff-cbc4f9532c53d8e89750066d832033bdR4)! :smirk:

Comment (sindresorhus) [inline build.js]: @kemitchell Exactly the same size:

``` js
got('spdx.org/licenses/licenses.json', {json: true}, (err, json) => {})
```

``` ruby
json = JSON.parse(Net::HTTP.get('spdx.org', '/licenses/licenses.json'))
```

;)

File: build.js (hunk 1 above)
Comment (shinnn) [inline build.js]: > sorting would keep the diff short

There seems to be almost no difference in diff size. Could you remove `.sort()`?

File: build.js (hunk 3 above)
Comment (shinnn) [inline build.js]: Yeah I prefer `got` :+1:

**Comment by kemitchell** [comment]
The things I do to make you people happy :wink:

**Comment by shinnn** [comment]
Thanks again, @kemitchell!

**Comment by kemitchell** [comment]
Thank you for the package! Let me know if I should bump my semver range in `spdx.js`.

**Comment by shinnn** [comment]
First, I should release a new version of this package.

https://github.com/shinnn/spdx-license-ids/commit/deab5ea0af7636ec7d14993358786e83fe57c042#diff-2cdbe42a14271da2b4b68477dada6be9

It seems that `UPL-1.0`, `RSA-MD` and `CNRI-Jython` has been removed. @kemitchell, do you think this change requires major version bump?

**Comment by kemitchell** [comment]
I still have no idea how to version packages that reflect changes to external data sets. @sindresorhus and I have gone back and forth on the idea a little bit.

I think this situation is a little unique, since with the patch, you're essentially republishing the official JSON list into npm. It strikes me as highly unlikely that there will be major "software" changes in how that gets presented as a module. But the SPDX license list isn't really versioned, as is the SPDX specification. To the extent you want to track it, you'd end up using dates as patch numbers, I guess.

If you'd like some practical, rather than theoretical or principled, input, I can run a search on the whole of npm to see if anyone is using the identifiers for those licenses. I strongly suspect the answer will be no. In any event, I have everything set up to run the checks. But it's getting late here in Oakland.

**Comment by shinnn** [comment]
> I strongly suspect the answer will be no.

I agree :+1: 

OK, I'll publish the next version as `1.0.1` and you don't need to change the semver range in `spdx.js` from `^1.0.0`.

**Comment by shinnn** [comment]
https://github.com/shinnn/spdx-license-ids/commit/05c7466fcd62c8642006ef354c95064fcade3a03

**Comment by kemitchell** [comment]
Thanks, @shinnn!

**Pull request merged**